utilities/src/test/java/org/craftercms/commons/file/blob/impl/AbstractBlobStoreTest.java

```diff
@@ -57,7 +58,7 @@ public void setUp() {
     @Test
     public void initTest() throws IOException, org.craftercms.commons.config.ConfigurationException {
         try (InputStream is = CONFIG_FILE.getInputStream()) {
-            HierarchicalConfiguration<ImmutableNode> config = ConfigUtils.readXmlConfiguration(is, ',' ,emptyMap());
+            HierarchicalConfiguration<ImmutableNode> config = ConfigUtils.readXmlConfiguration(is, ',' ,emptyMap(), emptyMap());
```

Comment:
Should we not add a test case for the new code?

Reply:
Agree with @sumerjabri, maybe we need more unit tests especially because of the changes to defaults

Reply:
Added a few test cases